addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -495,11 +495,9 @@
         <t t-foreach="challenges" t-as="challenge">
             <t t-set="challenge_done" t-value="challenge in challenges_done if challenges_done else False"/>
             <div t-attf-class="d-flex mb-3 align-items-center #{'o_wslides_entry_muted' if not challenge_done else ''}">
-                <img class="me-2"
-                    style="max-height: 36px;"
-                    t-att-src="website.image_url(challenge.reward_id, 'image_128') if challenge.reward_id.image_1920 else
-                               '/website_profile/static/src/img/badge_%s.svg' % (challenge.reward_id.level)"
-                    t-att-alt="challenge.reward_id.name"/>
+                <img t-if="not challenge.reward_id.image_1920" t-attf-src="'/website_profile/static/src/img/badge_%s.svg' % (challenge.reward_id.level)" t-att-alt="challenge.reward_id.name" style="max-height: 36px" class="me-2"/>
```

Comment:
Same idea here, would switch those around, show an image if available, otherwise use the fallback.